src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -256,7 +256,7 @@
             consumes = APPLICATION_JSON_VALUE,
             produces = APPLICATION_JSON_VALUE
     )
-    @Secured("cwd-admin")
+    @Secured({"cwd-system-user"})
```

Comment:
The ticket says to add below role as well
`cwd-read-only`. Correct me if I am wrong

Reply:
For now, we need the system user role only.